draft-ietf-tls-tls13.md

```diff
@@ -5204,7 +5226,7 @@ applicable features:
 
 A client is considered to be attempting to negotiate using this
 specification if the ClientHello contains a "supported_versions"
-extension with 0x0304 as the highest version number contained in its body.
+extension with 0x0304 contained in its body.
```

Comment:
This one I could go either way on, but it seems that if you're going to allow 1.3 to be negotiated, you still need to provide what 1.3 will need.

Reply:
Concur.